app/components/Settings/Settings.react.js

```diff
@@ -113,15 +113,25 @@ export default class Settings extends Component {
             );
 
             step3 = (
+                /*eslint-disable */
                 <div className={styles.step3Container}>
                     <h5>3. Query from Plotly 2.0</h5>
-                    <div>
-                        Query data by clicking on 'import data' on
-                        plot.ly/alpha/workspace and choose the SQL option.<br/>
+                    <div className={styles.futureDirections}>
+                        Query data by clicking on 'import data' from
+                        {" "}
+
+                        <a href="javascript:require('electron').shell.openExternal('https://plot.ly/alpha/workspace');">
```

Comment:
ye for sure, i found the quick solution to opening an external shell in a github issue and forgot to convert it to es6 and overall better look. l'll change it right away.